autotst/conformer/systematic.py

```diff
@@ -105,14 +106,18 @@ def opt_conf(i):
         """
         A helper function to optimize the geometry of a conformer.
         param i: index of the conformer
-        """   
-        conformer = conformers[i] #use the global object
+        """
+        try:   
+            conformer = conformers[i] #use the global object
+        except NameError:
```

Comment:
@nateharms can you elaborate on this? Why `NameError` instead of `KeyError`?

Reply:
Was doing `NameError` because when testing the `opt_conf` function, it would require a global `conformers` dict however I was having trouble creating one of those for a unit test. Because of this, I gave it `NameError` in the case when a `conformers` dict isn't available. I have now modified it to be a general `except:` to catch both Key- and NameErrors.